go.mod

```diff
@@ -7,18 +7,19 @@ require (
 	github.com/hashicorp/go-version v1.2.0
 	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.0.10
 	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.13.0
-	github.com/openshift/api v0.0.0-20210713130143-be21c6cb1bea // indirect
-	github.com/openshift/client-go v0.0.0-20200320143156-e7fa42a1261e
-	github.com/spf13/cobra v1.1.3
+	github.com/onsi/gomega v1.14.0
+	github.com/openshift/api v0.0.0-20210924154557-a4f696157341
+	github.com/openshift/client-go v0.0.0-20210916133943-9acee1a0fb83
```

Comment:
this is fine. Do we need to take steps to make sue this is compatible with the k8s client-go we are using?

Reply:
it updated automatically by `go mod tidy` so I think it good to go, otherwise you will have errors under func or unit tests

Reply:
works for me!